docs/installation.rst

```diff
@@ -30,8 +30,9 @@ Additional Prerequisites
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
 1. `CMake <https://cmake.org/>`__ version 3.18 or later
-2. `pyTorch <https://pytorch.org/>`__ with GPU support
-3. `Ninja <https://ninja-build.org/>`__
+2. `Ninja <https://ninja-build.org/>`__
+3. [For pyTorch support] `pyTorch <https://pytorch.org/>`__ with GPU support
+3. [For JAX support] `JAX <https://github.com/google/jax/>`__ with GPU support
```

Comment:
Since we know that not every version of JAX is going to work, could we add the specific version number here?

Reply:
Done.